src/barebones_mpc/model/inv_pendulum_model.py

```diff
@@ -0,0 +1,56 @@
+from src.barebones_mpc.model.abstract_model import AbstractModel
+import numpy as np
+
+class InvPendulumModel(AbstractModel):
+    def __init__(self, time_step, number_samples, sample_length, cart_mass, pendulum_mass):
+        """ Inverted pendulum model. The states are [y, v, theta, q], input is [u]
+        y: cart position
+        v: cart velocity
+        theta: pendulum angle
+        q: pendulum angle rate of change
+        u: force applied to the cart
+
+        """
+        self.input_dimension = 1
+        self.state_dimension = 4
+        self.time_step = time_step
```

Comment:
Are we going to expose `time_step` as a parameter in every model. If so we are going to make a small change in the __init__ of the baseclass and every child class